assets/airports/lrop.json

```diff
@@ -0,0 +1,1150 @@
+{
+    "radio": {
+        "twr": "Otopeni Tower",
+        "app": "Bucharest Approach",
+        "dep": "Bucharest Radar"
+    },
+    "icao": "LROP",
+    "iata": "OTP",
+    "magnetic_north": 5,
+    "ctr_radius": 180,
+    "ctr_ceiling": 17500,
+    "initial_alt": 6000,
+    "position": ["N44d34m16.00", "E26d05m06.00", "314ft"],
+    "rr_radius_nm": 5,
+    "rr_center": ["N44d34m16.00", "E26d05m06.00"],
+    "has_terrain": true, 
+    "wind": {
+        "angle": 90,
+        "speed": 10
+    },
+    "airspace": [{
```

Comment:
The division of the airspace isn't quite supported yet so the scope loses some transparency. For now the best solution is to comment this section (simply replace `airspace` with `_airspace`), and add the outer perimeter of the airspace inside the `airspace` key